examples/annotation/flows/detailed_annotation/prompt.yaml

```
@@ -0,0 +1,30 @@
+    Category: Sports
+
+    Text: A Cosmic Storm: When Galaxy Clusters Collide Astronomers have found what they are calling the perfect cosmic storm, a galaxy cluster pile-up so powerful its energy output is second only to the Big Bang.
+    Category: Sci/Tech
+
+    Text: Economy builds steam in KC Fed district The economy continued to strengthen in September and early October in the Great Plains and Rocky Mountain regions covered by the Tenth Federal Reserve District, the Federal Reserve Bank of Kansas City said Wednesday.
+    Category: Economy
+
+    Here is the query for annotation:
+
+    Text: {{text}}
+    Category: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)